pkg/status/render.go

```diff
@@ -56,29 +56,53 @@ func FormatStatus(data []byte) (string, error) {
 	snmpTrapsStats := stats["snmpTrapsStats"]
 	title := fmt.Sprintf("Agent (v%s)", stats["version"])
 	stats["title"] = title
-	renderStatusTemplate(b, "/header.tmpl", stats)
-	renderChecksStats(b, runnerStats, pyLoaderStats, pythonInit, autoConfigStats, checkSchedulerStats, inventoriesStats, "")
-	renderStatusTemplate(b, "/jmxfetch.tmpl", stats)
-	renderStatusTemplate(b, "/forwarder.tmpl", forwarderStats)
-	renderStatusTemplate(b, "/endpoints.tmpl", endpointsInfos)
-	renderStatusTemplate(b, "/logsagent.tmpl", logsStats)
-	if config.Datadog.GetBool("system_probe_config.enabled") {
-		renderStatusTemplate(b, "/systemprobe.tmpl", systemProbeStats)
+
+	headerFunc := func() { renderStatusTemplate(b, "/header.tmpl", stats) }
```

Comment:
nit: to ease unit-testing

we can imaging to have instead of `func()` a `func(b bytes.Buffer, stats map[string]interface{})`

```suggestion
	headerFunc := func(buffer bytes.Buffer, iStats map[string]interface{}) { renderStatusTemplate(buffer, "/header.tmpl", iStats) }
```
and for function like `forwarderFunc` 

```golang
forwarderFunc := func(buffer bytes.Buffer, iStats map[string]interface{})  { renderStatusTemplate(b, "/forwarder.tmpl", stats["forwarderStats"]) }
```
and so you can define it as standard function

```golang
func	renderHeader(buffer bytes.Buffer, iStats map[string]interface{}) { renderStatusTemplate(buffer, "/header.tmpl", iStats) }
//...
```